Asterisk.2013/Asterisk.NET/Manager/ManagerReader.cs

```diff
@@ -133,9 +133,9 @@ private void mrReaderCallbback(IAsyncResult ar)
 				// \n - because not all dev in Digium use \r\n
 				// .Trim() kill \r
 				lock (((ICollection) lineQueue).SyncRoot)
-					while (!string.IsNullOrEmpty(mrReader.lineBuffer) && (idx = mrReader.lineBuffer.IndexOf('\n')) >= 0)
-					{
-						line = idx > 0 ? mrReader.lineBuffer.Substring(0, idx).Trim() : string.Empty;
+                    while (!string.IsNullOrEmpty(mrReader.lineBuffer) && (((idx = mrReader.lineBuffer.IndexOf('\n')) >= 0) || ((idx = mrReader.lineBuffer.IndexOf("\n")) >= 0)))

```

Comment:
Why did you undo the `'\n'` change?
Are you forking from the latest master?

Reply:
Yeah it's the latest.
"From technical side I had a problem with the connection on docker linux env
themicster answer helped with the issue so I added the '\n' at ManagerReader ln:136 and the problem gone."

Reply:
And here you are removing that fix.

Reply:
Didn;t notice it was fixed 😕

Reply:
Ok you weren't undoing the edit, I didn't see the ||. But doing those IndexOf("\n") and IndexOf('\n') would be silly. And even so, please don't do multiple different unrelated things in the same pull request.